tests/parametric/test_otel_span_methods.py

```diff
@@ -396,6 +398,212 @@ def test_otel_set_attributes_separately(self, test_agent, test_library):
             assert span["name"] == "kafka.receive"
             assert span["resource"] == "operation"
 
+    @missing_feature(context.library < "java@1.26.0", reason="Implemented in 1.26.0")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(context.library == "python", reason="Not implemented")
+    @missing_feature(context.library == "python_http", reason="Not implemented")
```

Comment:
`python_http` was renamed to `python`. The original `python` grpc client was removed (we are planning to convert all grpc parametric endpoints to http).

```suggestion
    @missing_feature(context.library == "python", reason="Not implemented")
```